### PR TITLE
Validate tokenEndpointAllowReusePvtKeyJwt on form submit

### PR DESCRIPTION
**Pull request by mpmadhavig**
### Purpose
$subject

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [x] UX/UI review done on the final implementation.
- [x] Documentation provided. (Add links if there are any)
- [x] Unit tests provided. (Add links if there are any)
- [x] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [x] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: .changeset/gorgeous-lies-begin.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/console": patch
```
Comment (Achintha444) [inline .changeset/gorgeous-lies-begin.md:2]: bump `admin.applications.v1` as well

Comment (mpmadhavig) [reply .changeset/gorgeous-lies-begin.md:2]: Fixed with 31c268c3ca49e1e7a91773f96f56e3d0f2931ec6

**Review by pavinduLakshan — APPROVED** [review]
Better if we can add a unit test for this specific case to detect this type of issues from the pr level.

**Comment by mpmadhavig** [comment]
> Better if we can add a unit test for this specific case to detect this type of issues from the pr level.

This fix is more of the temporary fix for this form submission with non-updated value issue. May be we can add unit tests with the proper fix.

**Pull request merged**